.nais/poao-nais-dev.yaml

```diff
@@ -87,7 +87,8 @@ spec:
           },
           "header": {
             "csp": {
-              "connectSrc": ["'self'", "wss:", "*.nav.no", "*.adeo.no", "cdn.sanity.io", "storage.googleapis.com"],
+              "scriptSrc": ["'self'", "blob:", "unsafe-inline", "unsafe-eval", "*.nav.no", "*.adeo.no", "*.google-analytics.com", "*.googletagmanager.com", "account.psplugin.com", "*.hotjar.com"],
```

Comment:
Trenger man alle disse her altså? `unsafe-` høres jo litt sketchy ut 😅

Reply:
Når man ikke spesifiserer scriptSrc arver man vel konfigurasjonen fra poao-frontend? Vi har behov for å legge til self og blob, og må da også legge inn det som ligger i scriptSrc i poao-frontend for å ikke brekke noe(?)